docs/proposals/keadm-debug.md

```diff
@@ -0,0 +1,316 @@
+---
+title: KubeEdge debug (Issue 324)
+status: Pending
+authors:
+    - "@shenkonghui"
+    - "@qingchen1203"
+approvers:
+    - 
+creation-date: 2020-07-15
+last-updated:  2020-07-16
+---
+
+# Motivation
+
+Many users shared their feedback that kubeEdge edge nodes do not have good debugging and diagnostic methods, which may prevent people from trying kubeEdge.
+There should be a simple and clear way to help operation and maintenance personnel to ensure the stable operation of kubeEdge, so that users can focus more on using it immediately.
+
+# Proposal
+
+KubeEdge should have simple commands to debug and troubleshoot edge components.
+Therefore, it is recommended to use the following commands during operation and maintenance of KubeEdge.
+
+## Inscope
+
+1. To support first set of basic commands (listed below) to debug edge (node) components.
+
+For edge, commands shall be:
+
+- `keadm help`
+  - `keadm debug diagnose`
+  - `keadm debug collect`
+  - `keadm debug check`
+  - `keadm debug get`
+
+# Scope of commands
+
+## Design of the commands
+
+**NOTE**: All the below steps are executed as root user, to execute as sudo user. Please add `sudo` before all commands.
+
+### keadm debug  --help
+
+```
+keadm help command provide debug function to help diagnose the cluster
+
+Usage:
+  keadm debug [command]
+
+Examples:
+
+Available Commands:
+  diagnose        Diagnose specific fault scenarios in an all-round way and locate the cause of the fault
+  collect         Obtain all data of the current node, and then locate and use operation personnel
```

Comment:
what dose it mean?

Reply:
Obtain all the data of the current node, and provide it to the operation and maintenance personnel to locate the problem.  

Maybe translation error.

Reply:
> what dose it mean?

Obtain all the data of the current node, and provide it to the operation and maintenance personnel to locate the problem.

Maybe translation error.